Model/Service/DatabaseRule.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Flancer32\Csp\Model\Service;
+
+use Flancer32\Csp\Model\ResourceModel\Fl32RuleSent;
+use Flancer32\Csp\Api\Repo\Data\Rule as ERule;
+
+class DatabaseRule extends \Flancer32\Csp\Ui\DataProvider\Rule\Grid
```

Comment:
`DatabaseRule` seems a weird name. Does it describe what the service is responsible for? And why does it extend a UI grid class?

Reply:
It used to access the queries, which were refactored into the resource.The inheritence was no longer necessary, but forgotten